scenarios/rds_monitoring/infrastructure.tf

```diff
@@ -0,0 +1,54 @@
+locals {
+  workspace_prefix = terraform.workspace == "default" ? "" : terraform.workspace
+  key_pair = {
+    public_key = var.public_key
+    key_name   = "${local.workspace_prefix}_kp_${var.scenario}"
+  }
+}
+
+module "postgresql" {
+  source = "../modules/postgresql"
+
+  availability_zone = var.availability_zone
+  instance_name     = "${var.scenario}_db_${local.workspace_prefix}"
+
+  network_id  = var.network_id
+  router_id = var.router_id
+  subnet_id   = var.network_id
+  subnet_cidr = "${var.addr_3}.0/24"
```

Comment:
you have variable `subnet_cidr`

Reply:
subnet_cidr is 192.168.0.0

Reply:
well, then remove pls changes in `csm_controller`